sdks/wasm/src/library_mono.js

```diff
@@ -41,9 +41,15 @@ var MonoSupportLib = {
 				var decoder = typeof TextDecoder !== 'undefined' ? new TextDecoder ('utf-16le') : undefined;
```

Comment:
Would it be valid to cache this decoder? Seems that it has some impact https://jsben.ch/VfCSJ

Reply:
It appears to be stateless, so I'm in favor of caching it.

Reply:
Yeah it should be fine

Reply:
It's now build at startup and used conditionally.